### DISCUSSION TITLE
Updating/upgrading a raspbian OVOS

### DISCUSSION THREAD

**Discussion by wita-sec** [OpenVoiceOS/ovos-buildroot#184] [Q&A] [answered]
Dear community,

I consider to move from Picroft to OVOS. 
Looking through the installation guide for the raspbian headless image I am wondering,
whether the image, once it has been installed and booted, can be updated/upgraded with a newer version of OVOS,
and which would be the steps to follow.

Thank you for your advice.

Michael

**Comment by NeonDaniel** [comment]
I don't believe any of the current OVOS images have an official upgrade process, but there are some shell scripts that can be used to update python packages. @builderjer might know better than I about that

**Comment by builderjer** [comment]
There is no real official way, as of yet, but there have been some scripts made.  They have not been updated for a while, but you should be able to modify them to what you need.  
https://github.com/OpenVoiceOS/scripts

**Comment by wita-sec** [comment]
Thank you our comments and your pointer to the scripts. 
BTW, do you maintain a change log and if so please can you point me to that?

**Reply by builderjer** [reply, marked as answer]
No, this image is still very WIP and doesn't always work completely.  When I get a stable image, which it is close, I will do a changelog.  The main thing is, every image uses the latest git packages of ovos.  When ovos-core 0.1.0 is out, I will do a stable image and go from there.

On a side note, this is just a hobby for me that I wish I had more time for.  I Would love feedback, issues, and especially PR's

**Reply by builderjer** [reply]
could probably automate one, but I haven't had much experience with github actions

**Comment by wita-sec** [comment]
Your efforts are very much appreciated. I will recomment OVOS.  

I had a look into the documentation chapter "install OVOS as a library". At this stage, it appears to be more flexible compared to working with a pre-built image. 
I will give this a try, based on the OVOS-Core 0.1.0 stack.
